src/extensionFunctions.ts

```diff
@@ -183,6 +192,19 @@ export namespace extensionFunctions {
 		sendTextToOutputChannel(`Sent terminal ${name} the text ${text}`);
 	}
 
+	// send a message to a named terminal
```

Comment:
the comment seems not in phase to what the method is doing. this is not "a message" that is sent be a "Ctrl+C"
I would suggest to remove it.